src/content/docs/style-guide/writing-docs/processes-procedures/create-preview-document.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Create content for the preview docs site
+contentType: page
+template: basicDoc
+topics:
+  - Basic style guide
+  - Writing guidelines
+redirects:
+---
+
+The preview docs site hosts preview docs for upcoming features that we share with customers.
+
+Unlike the public docs site, the preview docs site is not publicly editable and does not appear in search engine results. Additionally, the site is not indexed, so you can't navigate from one preview doc to another. Because the preview docs site is located in a different repo than the public docs site, contributing looks a little different.
+
+Follow the steps below to create and edit preview docs.
+
+## Set up your local environment [#do-first]
+
+1. Go to the [`beta-docs-site repo`](https://github.com/newrelic/beta-docs-site). If you do not have access, contact the #doc-eng-help Slack channel for help.
+2. Clone the beta-docs-site repo on your computer.
+3. Make sure you're working in the correct repo on Github Desktop.
+4. Create a branch off the `main` branch.
+  
+## Create a new preview doc [#create-docs]
+
```

Comment:
This could use an intro sentence clarifying that, for the most part, the formatting and syntax of the content is the same as we use on the public docs site.